lib/binance/futures/rest/http_client.ex

```diff
@@ -0,0 +1,160 @@
+defmodule Binance.Futures.Rest.HTTPClient do
+  @endpoint "https://fapi.binance.com"
+
+  def get_binance(url, headers \\ []) do
+    HTTPoison.get("#{@endpoint}#{url}", headers)
+    |> parse_response
+  end
+
+  def delete_binance(url, headers \\ []) do
+    HTTPoison.delete("#{@endpoint}#{url}", headers)
+    |> parse_response
+  end
```

Comment:
If use pipe, can we use pipe all the way. It's the convention

```elixir
"#{@endpoint}#{url}"
|> HTTPoison.delete(headers)
|> parse_response
```

Please apply this style to all other places also. We can apply credo for this later on

Reply:
👍